docs/index.md

```diff
@@ -717,6 +726,71 @@ The following example is the procedure to generate a node from the function node
 
    -> You can use format-date node on your Node-RED flow editor.
 
+
+<a name="how-to-create-a-node-from-subflow"></a>
+
+## How to create a node from subflow
+
+After creating a subflow, add module properties of the subflow template.  Then, download JSON format subflow definition using "Export" menu of Node-RED editor.
+
+(1) Export JSON format of subflow
+
+   We assume node-red-contrib-qrcode as the module name and SUBFLOW JSON data is downloaded to a file named qrcode.json.
+ます。
+
+(2) Generate node using node-red-nodegen command
+
+    node-red-nodegen qrcode.json
+
+Node-RED users typically import generated node to the palette of Node-RED flow editor using the following procedures.
+
+(3) Change the current directory to generated node's directory
+
+    cd node-red-contrib-qrcode
+
+(4) Install depending libraries
```

Comment:
Again - as per above I think you can just go into your .node-red directory then `npm install {path-to}/node-red-contrib/qrcode` and the install and linking will happen automatically.